README.md

```diff
@@ -1,28 +1,37 @@
-# bedmess
+# BEDMS
 
-bedmess is a tool used to standardize genomics/epigenomics metadata based on a schema chosen by the user ( eg. ENCODE, FAIRTRACKS).
+BEDMS (BED Metadata Standardizer) is a tool used to standardize genomics/epigenomics metadata based on a schema chosen by the user ( eg. ENCODE, FAIRTRACKS, BEDBASE).
 
 
-To install `attribute-standardizer` , you need to clone this repository first. Follow the steps given below to install:
+## Installation
 
+To install `bedms` use this command: 
 ```
-git clone https://github.com/databio/bedmess.git
-
-cd bedmess
-
-pip install .
-
+pip install bedms
+```
+or install the latest version from the GitHub repository:
+```
+pip install git+https://github.com/databio/bedms.git
 ```
 
 ## Usage
```

Comment:
Is there a need for more documentation than this at the moment? If so, where will it live? The Bedbase docs repository? We could also put the changelog there as well.